LONG_DESCRIPTION.rst

```diff
@@ -15,6 +15,6 @@ You can install this package by using the pip tool and installing:
 Using Ably for Python
 ---------------------
 
-- Sign up for Ably at https://www.ably.io/
+- Sign up for Ably at https://ably.com/sign-up
 - Get usage examples at https://github.com/ably/ably-python
-- Visit https://www.ably.io/documentation for a complete API reference and more examples.
+- Visit https://ably.com/documentation for a complete API reference and more examples.
```

Comment:
While you're here, I notice the other bullets don't have a terminating full stop, so I would remove the one from this line also.

```suggestion
- Visit https://ably.com/documentation for a complete API reference and more examples
```

Reply:
Spoke to Quintin about this and am resolving it (Owen has kindly rubber-stamped the requested changes).